### ISSUE TITLE
Error after searching for saved hood

### ISSUE THREAD

**Issue by leekinney** [whathood/whathood#32]
I created a neighborhood and saved it then tried to search for it by name and received error. See screenshot. 
![screen shot 2015-08-19 at 7 37 51 pm](https://cloud.githubusercontent.com/assets/4022077/9372216/f5e6ccc0-46a9-11e5-9ac8-6009cbc2da1f.png)

**Comment by jimRsmiley** [comment]
@leekinney what was the name you searched for? It'd be awesome if you could get the url that you searched for. I tried:

http://whathood.in/search?q=LK-test

The neighborhood you added, but it doesn't create that exception.

**Comment by leekinney** [comment]
@jimRsmiley I'm not getting that error anymore either. It's still not showing the borders I drew though. I'm going to try to create another one and search and see what happens.

**Comment by leekinney** [comment]
@jimRsmiley Yep. Same results. I created another test hood and when I searched for it didn't get an error, just a blank screen http://whathood.in/search?q=leeann+test

**Comment by leekinney** [comment]
When I searched for a neighborhood near my test it popped up but still won't show me the borders I drew after clicking on it. 
![screen shot 2015-08-20 at 6 16 11 pm](https://cloud.githubusercontent.com/assets/4022077/9397266/c61703e2-4768-11e5-9b9f-58d486dde398.png)

**Comment by leekinney** [comment]
received error again after creating new hood searching for it and clicking on it
![screen shot 2015-08-20 at 6 38 48 pm](https://cloud.githubusercontent.com/assets/4022077/9397482/3370d1b4-476b-11e5-849d-d3b193e7b845.png)
